os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_peripheral/ble_tizenrt_server.c

```diff
@@ -242,6 +242,47 @@ trble_result_e rtw_ble_server_charact_notify(trble_attr_handle attr_handle, trbl
     return TRBLE_SUCCESS;
 }
 
+trble_result_e rtw_ble_server_charact_indicate(trble_attr_handle attr_handle, trble_conn_handle con_handle, uint8_t *data_ptr, uint16_t data_length)
+{
+    debug_print("send indicate abs_handle 0x%x \n", attr_handle);
```

Comment:
It can be failed before sending successfully.
So I think it is better to move this log at the line after 270.

Reply:
the return value from ble_tizenrt_server_send_msg(BLE_TIZENRT_MSG_INDICATE, param) below indicates the success or fail of sending message. 
The debug log here is to print attribute handle received from upper API, and help to check handle value.